src/components/PersonPage/PersonContainer.stories.mdx

```diff
@@ -0,0 +1,31 @@
+import { action } from "@storybook/addon-actions";
+import { text, array, number, date } from "@storybook/addon-knobs";
+import { Meta, Props, Story } from "@storybook/addon-docs/blocks";
```

Comment:
Just FYI addon-knobs will be deprecated soon and we should stop using it to not create any more work for #21.

Also, I stopped writing stories in .mdx and started using .tsx (for example, this [story](https://github.com/CouncilDataProject/cdp-frontend/blob/main/src/components/Details/EventVideo/EventVideo.stories.tsx)). My editor doesn't provide any auto-suggestion for mdx files. You can still write .mdx file though.

Reply:
I'll start using .tsx instead, thanks for lookin' out.